R/model.R

```diff
@@ -519,16 +528,21 @@ model_emax_binary <- function(
   )
   class(mod) <- c("dreamer_emax_binary", "dreamer_binary")
   attr(mod, "type") <- "EMAX"
+  rlang::inform("Note updated EMAX model parameterization", 
+                .frequency = "once",
+                .frequency_id = "3")
   return(mod)
 }
 
 #' @rdname model
+#' @param bound is the upper truncation bound of the exponential scale parameter \eqn{b_3}
 #' @section Exponential Binary:
-#'   \deqn{y \sim Binomial(n, f(d))}
-#'   \deqn{link(f(d)) = b_1 + b_2 * (exp(b_3 * d) - 1)}
+#'   \deqn{y \sim N(f(d), \sigma^2)}
```

Comment:
This is incorrect -- we have a binary outcome so the likelihood is binomial.  Please revert.